src/main/java/com/sivalabs/springblog/post/domain/PostService.java

```diff
@@ -0,0 +1,38 @@
+package com.sivalabs.springblog.post.domain;
+
+import com.sivalabs.springblog.ApplicationProperties;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional
+public class PostService {
+    private final PostRepository postRepository;
+    private final ApplicationProperties properties;
+
+    public PostService(PostRepository postRepository, ApplicationProperties properties) {
+        this.postRepository = postRepository;
+        this.properties = properties;
+    }
+
+    public PagedResult<Post> getPosts(int pageNo) {
+        Sort sort = Sort.by("title").ascending();
```

Comment:
Usually posts are displayed in reverse chronological order(created_date desc). Could you please update Sort object reflecting that.